### PR TITLE
rdpsign: add page

### PR DESCRIPTION
**Pull request by pxgamer**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/windows/rdpsign.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Enable quiet output:
+
+`rdpsign {{path/to/file.rdp}} /q`
+
+- Display verbose warnings, messages and statuses:
+
+`rdpsign {{path/to/file.rdp}} /v`
+
+- Test the signing and output the result without updating the input:
```
Comment (agnivade) [inline pages/windows/rdpsign.md]: Maybe just say - "Test the signing by displaying the output to console without updating the file" ?

Comment (pxgamer) [reply pages/windows/rdpsign.md]: Just interested, is it better to say `output to console` or `output to stdout`?

Comment (sbrl) [reply pages/windows/rdpsign.md]: I'd probably go with `stdout` myself, as it makes sense in both interactive and non-interactive contexts.

Comment (agnivade) [reply pages/windows/rdpsign.md]: I'm fine with both. I think in other pages, we have a mixture of both console and stdout.

Comment (pxgamer) [reply pages/windows/rdpsign.md]: Went with `stdout` :+1:

**Pull request merged**